src/lib/env-vars.ts

```diff
@@ -153,6 +153,26 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 			},
 		},
 	},
+	{
+		capableDeviceTypes: [
+			'blackboard-tx2',
+			'jetson-nano',
+			'jetson-tx2',
+			'jn30b-nano',
+			'n510-tx2',
+			'orbitty-tx2',
+			'spacely-tx2',
+			'srd3-tx2',
+		],
+		properties: {
+			RESIN_HOST_EXTLINUX_fdt: {
+				type: 'string',
+				description:
+					'Define the DTB to be used. Only supported by supervisor versions >= v11.11.0.',
+				examples: ['/path/to/file'],
```

Comment:
```suggestion
				examples: ['default'],
```

Reply:
@20k-ultra Isn't this supposed to be a file path or a filename? If it's supposed to be a path as we said, then imo this doesn't communicate that good enough.

Reply:
I totally understand. @acostach brings up some good points on this which convinced me it was a good idea: 


_I believe we could "tegra186-quill-p3310-1000-c03-00-base.dtb" or simply "default", because on the jetson nano there is another name. Using "default" also covers the case where nvidia decides to rename the dtb name in a future release. Using default also shows that the dtb from the raw emmc  partition is used._

Perhaps we can use `Default` and communicate it's a file path by changing the description. What about something like:

```Define a file path for the DTB to be used. Only supported by supervisor versions >= v11.11.0.'```